flutter/lib/src/default_integrations.dart

```diff
@@ -147,6 +149,24 @@ class NativeSdkIntegration extends Integration<SentryFlutterOptions> {
   }
 }
 
+/// Integration that captures certain window and device events.
+/// See also:
+///   - [SentryWidgetsBindingObserver]
+///   - [WidgetsBindingObserver](https://api.flutter.dev/flutter/widgets/WidgetsBindingObserver-class.html)
+class WidgetsBindingIntegration extends Integration<SentryFlutterOptions> {
+  @override
+  FutureOr<void> call(Hub hub, SentryFlutterOptions options) {
+    // We don't need to call `WidgetsFlutterBinding.ensureInitialized()`
+    // because `FlutterSentry.init` already calls it.
+    WidgetsBinding.instance.addObserver(SentryWidgetsBindingObserver(
```

Comment:
it'd be nice to override the `close` method and do:

`WidgetsBinding.instance.removeObserver`